gateway/analytics_test.go

```diff
@@ -368,12 +379,12 @@ func TestAnalytics_Write(t *testing.T) {
 					t.Error("Error executing test case")
 				}
 
-				// let records to to be sent
+				// let records to be sent
 				ts.Gw.Analytics.Flush()
 
 				results := ts.Gw.Analytics.Store.GetAndDeleteSet(redisAnalyticsKeyName)
 				if len(results) != 2 {
-					t.Fatal("Should return 1 record: ", len(results))
+					t.Fatal("Should return 2 records: ", len(results))
```

Comment:
I know, not your code, but if you're modifying it, `assert.Len(t, results, 2)` replaces the `if{}` completely.